packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -858,7 +858,7 @@ const getEchartAxis = ({
     const leftAxisYId = validCartesianConfig.layout.flipAxes
         ? validCartesianConfig.layout?.xField
         : validCartesianConfig.eChartsConfig.series?.find(
-              (serie) => serie.yAxisIndex === 0,
+              (serie) => serie.yAxisIndex === 0 || !serie.yAxisIndex, // NOTE: yAxisIndex is undefined on first time adding a serie
```

Comment:
when getting the `leftAxisYId`, check if there's an element in the series that has a `yAxisIndex` 0 or undefined (if they're on the right axis, the value is always set on change of the html select)